assets/js/vue/FileManager.vue

```diff
@@ -352,6 +371,16 @@ export default {
 
         onHelpButtonClick: function () {
             this.showHelpPopup = true;
+        },
+
+        queueDone: function (message) {
+            if (this.filesRenamed > 0) {
```

Comment:
If this function is to set the isRenamedPopupVisible, we can do it from where it is being called

Reply:
You mean like: `this.isRenamedPopupVisible = this.filesRenamed > 0` ??

Reply:
I put in a code suggestion below